### ISSUE TITLE
Broken dark theme

### ISSUE THREAD

**Issue by e7d** [gnunn1/tilix#1869]
Linux: Manjaro 19.0.2 (Kyria)
Gnome: 3.36.0-1
Tilix: 1.9.1
VTE: 0.60
GTK: 3.24.14

After having installed then removed KDE Plasma alongside Gnome on my Manjaro, applications-wide dark theme does not have any effect on Tilix submenu and main window anymore.

I tried to reinstall Gnome, my themes and Tilix with the following:
```shell
sudo pacman -S gnome
sudo pacman -S manjaro-gnome-assets manjaro-gdm-theme manjaro-settings-manager
sudo pacman -S gtk-engine-murrine gtk-engines
sudo pacman -S tilix
```
_Note:_ Rebooting did not help

Here is the actual result:
![Screenshot from 2020-04-07 12-07-08](https://user-images.githubusercontent.com/1412033/78657943-ca89a780-78c9-11ea-9a2d-29a8fa8bb03e.png)

I also recorded a video of the behavior of different applications including Tilix while changing applications theme: https://www.youtube.com/watch?v=5-Zs1TcsGKQ

**Comment by iambumblehead** [comment]
I don't have anything too useful to add here _but_ I am using arch linux with the latest everything gnome3 on multiple machines and dark mode tilix works fine for me. The problem @e7d is describing is perhaps specifically reproduced by installing and removing KDE Plasma

**Comment by e7d** [comment]
Yeah you are right. After some digging (a couple hours actually), I found that it's a known issue:
https://www.reddit.com/r/gnome/comments/fn0ksn/broken_dark_theme_after_removing_kdeplasma/

In my case, simply removing these files then restarting the session fixed the issue:
- ~/.config/gtk-3.0/colors.css
- ~/.config/gtk-3.0/gtk.css

The Reddit thread says to comment the `color.css` import in `gtk3.css`, but since this import is the sole instruction, I removed both of the files and the result is clean.

Consider it a false issue.
